keystone_performance/load_test.py

```diff
@@ -304,6 +314,73 @@ def notify_done(self):
         self._done = True
 
 
+class UserInfo(object):
+    def __init__(self, args):
+        auth_url = '%s/v3/auth/tokens' % args.url
+
+        if args.user_domain_id:
+            user_domain_info = {'id': args.user_domain_id}
+        else:
+            user_domain_info = {'name': args.user_domain_name}
+        if args.project_id:
+            project_info = {'id': args.project_id}
+        else:
+            project_info = {'name': args.project_name}
+        if args.project_domain_id:
+            project_info['domain'] = {'id': args.project_domain_id}
+        else:
+            project_info['domain'] = {'name': args.project_domain_name}
+        auth_req_body = {
+            'auth': {
+                'identity': {
+                    'methods': ['password'],
+                    'password': {
+                        'user': {
+                            'name': args.username,
+                            'domain': user_domain_info,
+                            'password': args.password
+                        }
+                    }
+                },
+                'scope': {
+                    'project': project_info
+                }
+            }
+        }
+
+        headers = {'content-type': 'application/json'}
+        r = requests.post(auth_url, json=auth_req_body, headers=headers)
+        if r.status_code != 201:
+            raise Exception('Error: Failed to get keystone token for getting user info. Status code: ' + str(r.status_code))
```

Comment:
Keep the lines at < 80 chars so that reviewers can see all the code in the github ui.